src/operator/account/ConvertAccountToContractOverlay.tsx

```diff
@@ -0,0 +1,266 @@
+import React, {
+  ChangeEvent,
+  createRef,
+  CSSProperties,
+  FC,
+  RefObject,
+  useContext,
+  useState,
+} from 'react'
+import {
+  Overlay,
+  Gradients,
+  Alert,
+  ComponentColor,
+  IconFont,
+  ButtonBase,
+  ButtonShape,
+  Appearance,
+  Button,
+  ClickOutside,
+  ComponentSize,
+  Form,
+  Popover,
+  PopoverInteraction,
+  PopoverPosition,
+  ButtonRef,
+  ComponentStatus,
+  Columns,
+  Grid,
+  Input,
+} from '@influxdata/clockface'
+import DatePicker from 'src/shared/components/dateRangePicker/DatePicker'
+import {AccountContext} from 'src/operator/context/account'
+import moment from 'moment'
+
+const noOp = () => {}
```

Comment:
Do we need some user notification or feedback here - I see noOp being invoked only when there's an invalid user input. Should we be sending a notification (e.g. `dispatch(notify([notification]))) when that happens?

Reply:
We do actually have some user feedback because we set `ComponentStatus` to `Error`, but I do agree that we should have an error reason. However, `dispatch(notify([notification]))` will only show a temporary error, which isn't great for form-related errors from a UX perspective, since it's easy for the user to miss the error and it leaves them without context. I'm okay with doing that if it's standard practice for us, but I think it might be worth reviewing Material Design for how they recommend error handling because there's some good information there. My proposal would be to include an error below our form fields similar to the ones in the Text Field reference below so that the error exists in a useful context within the UI and is persistent until the error is resolved. Ideally this would be built into the `DatePicker` component, but I don't mind adding it in for this component specifically in the meantime. What are your thoughts on that?

Material Design links:
[Text Fields](https://material.io/components/text-fields#anatomy)
[Snackbars](https://material.io/components/snackbars#usage)
[Error Patterns (Archived)](https://material.io/archive/guidelines/patterns/errors.html)